src/main/scala/io/iohk/ethereum/jsonrpc/JsonMethodsImplicits.scala

```diff
@@ -0,0 +1,37 @@
+package io.iohk.ethereum.jsonrpc
+
+import akka.util.ByteString
+import io.iohk.ethereum.jsonrpc.JsonRpcController.{JsonDecoder, JsonEncoder}
+import io.iohk.ethereum.jsonrpc.Web3Service.{ClientVersionRequest, ClientVersionResponse, Sha3Request, Sha3Response}
+import org.json4s.{CustomSerializer, DefaultFormats, Formats, JsonAST}
+import org.json4s.JsonAST.{JArray, JString}
+import org.json4s.JsonDSL._
+import org.spongycastle.util.encoders.Hex
+
+import scala.util.{Success, Try}
+
+object JsonMethodsImplicits {
+
+  implicit val formats: Formats = DefaultFormats + ByteStringSerializer
+
+  implicit val web3_sha3 = new JsonDecoder[Sha3Request] with JsonEncoder[Sha3Response] {
+    override def decodeJson(params: JsonAST.JArray): Try[Sha3Request] =
+      Try(Sha3Request(params.arr.head.extract[ByteString]))
```

Comment:
Should we check here or somewhere else that the `ByteString` parameter has a `0x` prefix? It seems to be needed by https://github.com/ethereum/wiki/wiki/JSON-RPC#hex-value-encoding, and geth fails when asked for request such as `{"jsonrpc":"2.0","method":"web3_sha3","params":["68656c6c6f20776f726c64"],"id":64}`